JNIBridge.h

```diff
@@ -12,6 +12,12 @@ typedef void* jvoid; // make it possible to return void
 
 namespace jni
 {
+	struct CallbackOverrides
+	{
+		typedef jclass(*FindClass)(JNIEnv*, const char*);
+
+		FindClass findClass;
```

Comment:
I'd prefer the variable to be named FindClass (with uppercase F), and the callback type to FindClassFunc (or FindClassCallback)